Submitted By: Andrew Bradford <andrew@bradfordembedded.com>
Date: 2013-10-25
Initial Package Version: 1.2.9.2
Origin:	CLFS specific
Upstream Status: Not Applied
Description: 	Fixes build problems.

diff -Naur netplug-1.2.9.2.orig/lib.c netplug-1.2.9.2/lib.c
--- netplug-1.2.9.2.orig/lib.c	2013-10-25 16:23:21.270934264 -0400
+++ netplug-1.2.9.2/lib.c	2013-10-25 16:24:26.067667210 -0400
@@ -200,20 +200,6 @@
 }
 
 
-void
-__assert_fail(const char *assertion, const char *file,
-              unsigned int line, const char *function)
-{
-    do_log(LOG_CRIT, "%s:%u: %s%sAssertion `%s' failed",
-           file, line,
-           function ? function : "",
-           function ? ": " : "",
-           assertion);
-
-    abort();
-}
-
-
 /*
  * Local variables:
  * c-file-style: "stroustrup"
diff -Naur netplug-1.2.9.2.orig/netplug.h netplug-1.2.9.2/netplug.h
--- netplug-1.2.9.2.orig/netplug.h	2013-10-25 16:23:21.270934264 -0400
+++ netplug-1.2.9.2/netplug.h	2013-10-25 16:24:17.054324652 -0400
@@ -25,6 +25,7 @@
 #include <sys/socket.h>
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
+#include <time.h>
 
 #define NP_SCRIPT NP_SCRIPT_DIR "/netplug"
 
